src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Ui {
```

Comment:
Similar to Parser class, maybe you can consider converting this to a utility class with static functions?